objects/Fan-MadeAccessories.aa8b38.json

```diff
@@ -15,8 +15,7 @@
   },
   "ContainedObjects_order": [
     "ArkhamFantasy-PixelArtMini-Cards.e17c9e",
-    "DrawTokenButtonTooltipRenamer.cc77a8",
-    "WhimsicalsGenericDifficultySelector.05efb4",
+    "GenericDifficultySelector.8112ff",
```

Comment:
removed broken and obsolete tooltip renamer and renamed generic difficulty selector